README.md

```diff
@@ -53,6 +53,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - uses: actions/checkout@v3
+
```

Comment:
Is this step needed if we don't ever look at the code files locally? I think its okay to remove this.

```suggestion
```

Reply:
@zkoppert unfortunately it is, because it seems that `gh` errors out if it doesn't find a checked out git repo.

We could use something other than `gh`, but it was convenient to do as it's installed by default in the GitHub actions env.